src/components/Layout/Header.tsx

```diff
@@ -1,79 +1,34 @@
-'use client';
-
-import { useState } from 'react';
 import Link from 'next/link';
 
-import { useAuth } from '../../context/AuthContext';
 import { LogoIcon } from '../icons/LogoIcon';
 import { Routes } from '../../constants';
-import { LightSwitch } from '../LightSwitch';
-import { useAuthModal } from '../../context/AuthModalContext';
-import { Button } from '../Button';
-import { useEffectAsync } from '../../hooks/useEffectAsync';
-import { getOwnerRequests } from '../../api/messages';
-import { OwnerRequestType } from '../../types/Message';
+import { CustomLink } from '../CustomLink';
 import { UserControls } from './UserControls';
 
 type HeaderProps = {
   defaultTheme: string;
 };
 
-export const Header: React.FC<HeaderProps> = ({ defaultTheme }) => {
```

Comment:
When I redirect to profile page, the popup stays open

![CleanShot 2025-01-05 at 23 13 33](https://github.com/user-attachments/assets/adbe2cd9-78ec-483e-8d70-8d17aee2be5d)


After any item clicks that lead to redirect (so not the light/dark theme switch), we should close the popup